util/downloader/chunk_test.go

```diff
@@ -0,0 +1,118 @@
+package downloader
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateChunks(t *testing.T) {
+	tests := []struct {
+		contentLength int64
+		totalChunks   int64
+		expected      []*chunk
+	}{
+		{
+			contentLength: 181403648,
```

Comment:
These values should be random. We can address this in another PR.

Reply:
We can't test random we need known start and end range to check actual and excepted chunk list.